iAM-legacy/Simulation/Committed.cs

```diff
@@ -30,6 +30,10 @@ public class Committed
         int _resultType;
         //If this is set. The committed project is a ScheduledTreatment and not a true committed project.
         public string ScheduledTreatmentId { get; set; }
+        public string SplitTreatmentId { get; set; }
```

Comment:
These properties should also be made private and have the committed project injected into the appropriate constructors or accessed via ID.

Reply:
Added constructor to committed for split treatment